numpy/core/tests/test_overrides.py

```diff
@@ -248,6 +261,19 @@ def __array_function__(self, func, types, args, kwargs):
         with assert_raises_regex(TypeError, 'no implementation found'):
             dispatched_one_arg(array)
 
+    def test_where_dispatch(self):
+
+        class DuckArray:
+            def __array_function__(self, ufunc, method, *inputs, **kwargs):
+                return "overridden"
+
+        array = np.array(1)
+        duck_array = DuckArray()
+
+        result = np.std(array, where=duck_array)
```

Comment:
@sberg, I used `std` instead of `copyto` since I thought it would be simpler with only one argument. Is this ok? I can change it to `copyto` if you think that would be better. Also, do we think I put this test in the right class? I kept going back and forth on where to put this.

Reply:
Sure, was just trying to make sure that we handle `where=` similar in both overrdies.